expo/gbe/models/event.py

```diff
@@ -23,26 +23,28 @@ class Event(EventItem):
     from participant bids.
     '''
     objects = InheritanceManager()
-    title = CharField(max_length=128)
-    description = TextField()            # public-facing description
+    e_title = CharField(max_length=128)
+    e_description = TextField()            # public-facing description
     blurb = TextField(blank=True)        # short description
     duration = DurationField()
     notes = TextField(blank=True)  # internal notes about this event
     event_id = AutoField(primary_key=True)
-    conference = ForeignKey(
+    e_conference = ForeignKey(
```

Comment:
And these three are the other big changes - to separate biddable vs. event fields.